rdt-open-reader/src/main/java/io/ona/rdt/util/Utils.java

```diff
@@ -263,7 +265,11 @@ private UserAuthorizationVerificationTask(Context context) {
         @Override
         protected Void doInBackground(Void... voids) {
             boolean isUserAuthorized = syncUtils.verifyAuthorization();
-            if (!isUserAuthorized) {
+
+            Activity currentActivity = RDTApplication.getInstance().getCurrentActivity();
+            boolean isFormActivity = currentActivity instanceof RDTJsonFormActivity;
```

Comment:
you should still check that the current activity is not null

Reply:
now there is no need to check null that will redundant.

Reply:
why is that? the not all activities register themselves with the application.

Reply:
i mean if the currentActivity is null then `isFormActivity` will set to false and if I add a null check then the IDE shows a warning like there is no need to check the `null` condition.